CONTRIBUTING.md

```diff
@@ -1,86 +1,81 @@
 $ git clone <repo-link>
 ```
-Now you can create separate branches in the forked repository and prepare pull requests with changes.
+
+Now, you can create separate branches in the forked repository and prepare pull requests with changes.
+
 ### Issues solving
-To solve issues, you can follow the steps below:
-1. Issue filed (by any user).
-2. Proper label assigned by gate-keeper.
-3. Bug-fixer forked and cloned.
-4. Optional clarifications made using the Issues tab's Comment section.
-5. Pull request with fix created.
-6. Pull request reviewed by others.
-7. All code review comments handled.
-8. Pull request accepted by gate-keeper.
-9. Pull request merged successfully.
+
+To solve issues, follow the steps below:
+
+1. A user (any user) files an issue.
+1. A gate-keeper assigns a proper label to the issue.
+1. A user who wants to fix the issue, the bug fixer, forks and clones the repository.
+1. Optional: the bug fixer asks for and receives clarifications using the Issues tab's Comment section.
+1. The bug fixer creates a pull request that fixes the issue.
+1. Others review the pull request.
+1. The bug fixer handles all code review comments.
+1. A gate-keeper accepts the pull request.
+1. The pull request merges successfully.
 
 ### Code review
-The code review process is designed to catch both style and domain specific issues. It is also designed to follow and respect the _definition of done_. Please make sure your code follows the style of the source code you are modifying. Each pull request must be reviewed by the gate-keeper before we can merge it to the master branch.
+
+The code review process catches both style and domain-specific issues. It also follows and respects the _definition of done_. Please make sure your code follows the style of the source code you are modifying. A gate-keeper must review each pull request before we can merge the pull request to the master branch.
+
 ## Issues and bug reporting
-Please report all bugs using the Issues tab on the GitHub web-page. It will help us to collaborate on issues more promptly. One of our gate-keepers (developers responsible for quality and the repository) will review the issue and assign a label such as _bug_, _enhancement_, _help wanted_ or _wontfix_.
-# How to contribute
-You can either file a bug, help fix a bug or propose a new feature (or enhancement) and implement it yourself. If you want to contribute please:
-* Bug reports: File a bug report in the Issues tab of this repo to let us know there are problems with our code.
+
+Please report all bugs using the Issues tab on the GitHub page. This process helps us collaborate on issues promptly. One of our gate-keepers (developers responsible for quality and the repository) will review the issue and assign a label, such as _bug_, _enhancement_, _help wanted_ or _wontfix_.
+
+## How to contribute
+
+You can file a bug, help fix a bug or propose a new feature (or enhancement) and implement it yourself. If you want to contribute, please:
```

Comment:
I think this sentence ends a bit abruptly. Maybe say something like `If you want to contribute, please see below?` What do you think @AnotherButler?